en-US/mozorg/internet-health/open-innovation.lang

```diff
@@ -0,0 +1,249 @@
+
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/open-innovation/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/open-innovation/
+
+
+;Open Innovation: Vital to Internet Health
```

Comment:
Explain that this is the HTML title of the page, and the next one is the description. You can check existing .lang files to see similar comments.